service/src/main/kotlin/fi/espoo/evaka/attendance/RealtimeStaffAttendanceController.kt

```diff
@@ -205,6 +216,13 @@ class RealtimeStaffAttendanceController(private val accessControl: AccessControl
                         body.entries.mapNotNull { it.id }
                     )
                     body.entries.map { entry ->
+                        accessControl.requirePermissionFor(
```

Comment:
This doesn't need to be executed in a loop multiple times, because all required data is always available. Maybe it could be moved to right after the Unit.UPDATE_STAFF_ATTENDANCES check?